src/subscription/subscribe.js

```diff
@@ -156,16 +155,18 @@ function subscribeImpl(
 
   // Resolve the Source Stream, then map every source value to a
   // ExecutionResult value as described above.
-  return sourcePromise.then((resultOrStream) =>
+  return sourcePromise.then((resultOrStream) => {
     // Note: Flow can't refine isAsyncIterable, so explicit casts are used.
```

Comment:
Now that you have `boolean %checks(value instanceof AsyncIterable)` in `isAsyncIterable` is it still true?

Reply:
@IvanGoncharov I left a comment addressing this here: https://github.com/graphql/graphql-js/pull/2712#discussion_r455113122